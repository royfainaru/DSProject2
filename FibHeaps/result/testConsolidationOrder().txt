╮
├── ╭────────────────╮
│   │ Key: 4         │
│   ├────────────────┤
│   │ Rank: 2        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: (null)   │
│   │ Child: 6       │
│   ╰────────────────╯
│   ├── ╭───────────────╮
│   │   │ Key: 6        │
│   │   ├───────────────┤
│   │   │ Rank: 1       │
│   │   │ Marked: false │
│   │   │ Parent: 4     │
│   │   │ Next: 5       │
│   │   │ Prev: (null)  │
│   │   │ Child: 7      │
│   │   ╰───────────────╯
│   │   ╰── ╭───────────────╮
│   │       │ Key: 7        │
│   │       ├───────────────┤
│   │       │ Rank: 0       │
│   │       │ Marked: false │
│   │       │ Parent: 6     │
│   │       │ Next: (null)  │
│   │       │ Prev: (null)  │
│   │       │ Child: (null) │
│   │       ╰───────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 5        │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 4     │
│       │ Next: (null)  │
│       │ Prev: 6       │
│       │ Child: (null) │
│       ╰───────────────╯
├── ╭────────────────╮
│   │ Key: 2         │
│   ├────────────────┤
│   │ Rank: 1        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 1        │
│   │ Prev: 4        │
│   │ Child: 3       │
│   ╰────────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 3        │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 2     │
│       │ Next: (null)  │
│       │ Prev: (null)  │
│       │ Child: (null) │
│       ╰───────────────╯
╰── ╭────────────────╮
    │ Key: 1         │
    ├────────────────┤
    │ Rank: 0        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: 2        │
    │ Child: (null)  │
    ╰────────────────╯
