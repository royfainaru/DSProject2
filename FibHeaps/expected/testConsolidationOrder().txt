╮
├── ╭────────────────╮
│   │ Key: 1         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: 4        │
│   │ Child: (null)  │
│   ╰────────────────╯
├── ╭────────────────╮
│   │ Key: 2         │
│   ├────────────────┤
│   │ Rank: 1        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 4        │
│   │ Prev: 1        │
│   │ Child: 3       │
│   ╰────────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 3        │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 2     │
│       │ Next: 3       │
│       │ Prev: 3       │
│       │ Child: (null) │
│       ╰───────────────╯
╰── ╭────────────────╮
    │ Key: 4         │
    ├────────────────┤
    │ Rank: 2        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 1        │
    │ Prev: 2        │
    │ Child: 6       │
    ╰────────────────╯
    ├── ╭───────────────╮
    │   │ Key: 6        │
    │   ├───────────────┤
    │   │ Rank: 1       │
    │   │ Marked: false │
    │   │ Parent: 4     │
    │   │ Next: 5       │
    │   │ Prev: 5       │
    │   │ Child: 7      │
    │   ╰───────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 7        │
    │       ├───────────────┤
    │       │ Rank: 0       │
    │       │ Marked: false │
    │       │ Parent: 6     │
    │       │ Next: 7       │
    │       │ Prev: 7       │
    │       │ Child: (null) │
    │       ╰───────────────╯
    ╰── ╭───────────────╮
        │ Key: 5        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 4     │
        │ Next: 6       │
        │ Prev: 6       │
        │ Child: (null) │
        ╰───────────────╯
